inst/example_apps/dynamic_render_app/acme_dashboard.Rmd

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
I can't help but notice that this dashboard looks completely empty. Should there be some basic chart or table or something, using one of the usual base R datasets?